components/ingest-service/backend/elastic/migration.go

```
@@ -260,9 +259,7 @@ func (es *Backend) ReindexNodeStateA1(ctx context.Context, a1NodeStateIndexName
 		}
 	}
 
-	es.RefreshIndex(ctx, mappings.NodeState.Index)
-
-	return err
+	return es.RefreshIndex(ctx, mappings.NodeState.Index)
```

Wasn't sure if we wanted to ignore errors from this refresh or not.

---

So the reasoning here is that `err` would have already been returned if it was set? Makes sense.